### PR TITLE
refactor: framework renderers and docs

### PR DESCRIPTION
**Pull request by WoodNeck**
## Issue
#409

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,32 +8,31 @@
...
-<img src="https://img.shields.io/static/v1.svg?label=&message=TypeScript&color=294E80&style=flat-square&logo=typescript" />
-<a href="https://deepscan.io/dashboard#view=project&tid=3998&pid=5802&bid=46086"><img src="https://deepscan.io/api/teams/3998/projects/5802/branches/46086/badge/grade.svg" alt="DeepScan grade" /></a>
+  <img src="https://img.shields.io/static/v1.svg?label=&message=TypeScript&color=294E80&style=flat-square&logo=typescript" />
 </p>
 
 <h3 align="center">
-  <a href="https://naver.github.io/egjs-flicking/">Demo</a> / <a href="https://naver.github.io/egjs-flicking/release/latest/doc/index.html">Documentation</a> / <a href="https://naver.github.io/egjs/"><img height="20" src="https://naver.github.io/egjs/img/logo.svg"/> Other components</a>
+  <a href="https://naver.github.io/egjs-flicking/">Demo</a> / <a href="https://naver.github.io/egjs-flicking/docs/api/Flicking">Documentation</a> / <a href="https://naver.github.io/egjs/"><img height="20" src="https://naver.github.io/egjs/img/logo.svg"/> Other components</a>
```
Comment (mixed) [inline README.md:29]: How to manage version?

File: README.md [hunk 2]
```diff
@@ -153,7 +134,7 @@ var flicking = new eg.Flicking("#el", { collectStatistics: false });
 ## 🌐 Supported Browsers
 |<img width="20" src="https://simpleicons.org/icons/internetexplorer.svg" alt="IE" />|<img width="20" src="https://simpleicons.org/icons/googlechrome.svg" alt="Chrome" />|<img width="20" src="https://simpleicons.org/icons/firefoxbrowser.svg" alt="Firefox" />|<img width="20" src="https://simpleicons.org/icons/safari.svg" alt="Safari" />|<img width="20" src="https://simpleicons.org/icons/apple.svg" alt="iOS" />|<img width="20" src="https://simpleicons.org/icons/android.svg" alt="Android" />|
 |:---:|:---:|:---:|:---:|:---:|:---:|
-|10+|Latest|Latest|Latest|7+|4+|
+|9+(With polyfill)|Latest|Latest|Latest|7+|4+|
```
Comment (mixed) [inline README.md:137]: I think that In [our polyfill policy](https://oss.navercorp.com/egjs/work/issues/575#issuecomment-4277333), it seems that we need to define which polyfill to put.

**Review by mixed — APPROVED** [review]
Awesome!

File: src/renderer/Renderer.ts [hunk 3]
```diff
@@ -130,65 +128,78 @@ abstract class Renderer {
...
    * @param {number} index Index to insert new panels at<ko>새로 패널들을 추가할 인덱스</ko>
-   * @param {Flicking.ElementLike | Flicking.ElementLike[]} element A new HTMLElement, a outerHTML of element, or an array of both
-   * <ko>새로운 HTMLElement, 혹은 엘리먼트의 outerHTML, 혹은 그것들의 배열</ko>
+   * @param {any[]} elements An array of element or framework component with element in it<ko>엘리먼트의 배열 혹은 프레임워크에서 엘리먼트를 포함한 컴포넌트들의 배열</ko>
    * @return {Panel[]} An array of prepended panels<ko>추가된 패널들의 배열</ko>
    */
-  public insert(index: number, element: ElementLike | ElementLike[]): Panel[] {
+  public batchInsert(...items: Array<{
+    index: number;
+    elements: any;
```
Comment (daybrush) [inline src/renderer/Renderer.ts:153]: I think it would be better to infer it with the any[] type.

**Review by daybrush — APPROVED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (WoodNeck) [reply README.md:137]: Good, I think it only needs Promise polyfill but should be tested.

File: README.md (hunk 1 above)
Comment (WoodNeck) [reply README.md:29]: We can manage versions with Docusaurus, that link will automatically connect to the latest version

File: src/renderer/Renderer.ts (hunk 3 above)
Comment (WoodNeck) [reply src/renderer/Renderer.ts:153]: Thanks for checking that, I'll fix it.

Comment (mixed) [reply src/renderer/Renderer.ts]: It would be nice if an explanation of what polyfill to add was added.

File: README.md (hunk 2 above)
Comment (mixed) [reply README.md:137]: It would be nice if an explanation of what polyfill to add was added.

Comment (WoodNeck) [reply README.md:138]: I've tested it and only es6 Promise polyfill was needed to run in IE9+

**Pull request merged**